### PR TITLE
docs(k8s): version policy edits

### PR DESCRIPTION
**Pull request by tgenaitay**
### Your checklist for this pull request

- [x] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

- 1.29 release
- New support timeline at 14 months
- Simplified version information

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 1]
```diff
@@ -42,64 +42,61 @@ Before opening a support ticket related to cluster behavior or a Kubernetes comp
...
+| 1.28               | August 2023      | October 2024         | August 2023                  | February 2025*   | April 7, 2025*    |
+| 1.27               | April 2023       | June 2024            | April 2023                   | February 2025*   | April 7, 2025*    |
+| 1.26               | December 2022    | February 2024        | February 6, 2023             | February 2025    | March 6, 2025     |
+| 1.25               | August 2022      | October 2023         | February 6, 2023             | February 2025    | March 6, 2025     |
 | 1.24               | April 2022       | July 2023            | April 2022                   | April 2024       | May 15, 2024      |
 | 1.23               | December 8, 2021 | February 2023        | December 9, 2021             | December 2023    | February 15, 2024 |
 
+<Message type="important">
+* New support policy: from version 1.29 release, Scaleway's support window has shifted to 14 months.
+We're discontinuing extended support for outdated upstream Kubernetes releases to ensure you can always benefit from the latest features, bug fixes, enhanced support and maximum security.
```
Comment (nerda-codes) [inline containers/kubernetes/reference-content/version-support-policy.mdx:68]:
```suggestion
We are discontinuing extended support for outdated upstream Kubernetes releases to ensure you can always benefit from the latest features, bug fixes, enhanced support and maximum security.
```

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 2]
```diff
@@ -42,64 +42,61 @@ Before opening a support ticket related to cluster behavior or a Kubernetes comp
...
+Consequently, versions 1.27 & 1.28 will have corresponding end of support timelines.
```
Comment (nerda-codes) [inline containers/kubernetes/reference-content/version-support-policy.mdx:69]:
```suggestion
Consequently, versions `1.27` and  `1.28` will have corresponding end of support timelines.
```

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 3]
```diff
@@ -42,64 +42,61 @@ Before opening a support ticket related to cluster behavior or a Kubernetes comp
...
-**Main changes:**
 * This release consists of 60 enhancements. 18 of those enhancements are entering alpha, 29 are graduating to beta, and 13 are graduating to stable.
 
 ### 1.26
 * Official [release announcement](https://kubernetes.io/blog/2022/12/09/kubernetes-v1-26-release/)
-
-**Main changes:**
-* Eleven features graduated to GA
-* Twelve features deprecated or removed
+* 11 features graduated to GA, 12 features deprecated or removed
```
Comment (nerda-codes) [inline containers/kubernetes/reference-content/version-support-policy.mdx:91]: What do you mean by "graduated" ? If you mean "moved to GA", maybe you could use "delayed" or "available during GA" something like that?

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 4]
```diff
@@ -42,64 +42,61 @@ Before opening a support ticket related to cluster behavior or a Kubernetes comp
...
 
-* **Release:** Version is available for cluster creation and cluster upgrade.
+* **Official End of Life:** Upstream [end of maintenance support](https://endoflife.date/kubernetes).
+* **Scaleway Release:** Version is available for Kubernetes Kapsule and Kubernetes Kosmos cluster creation and cluster upgrade.
 * **Deprecation:** Version is removed from cluster creation option. Customers are informed via a support ticket that they are using a deprecated version of Kubernetes and are asked to update their clusters.
-* **End of support:** Version is not supported anymore, clusters will be automatically upgraded to the next supported minor version.
+* **End of Support:** Version is not supported anymore, clusters will be automatically upgraded to the next supported minor version.
+
+<Message type="note">
+When a minor version becomes unsupported, Scaleway will operate an upgrade to the latest patch version of the next supported minor version within the next 30 days. You will be notified via ticket.
```
Comment (nerda-codes) [inline containers/kubernetes/reference-content/version-support-policy.mdx:51]:
```suggestion
When a minor version becomes unsupported, Scaleway operates an upgrade to the latest patch version of the next supported minor version within the next 30 days. You will be notified via ticket.
```

File: containers/kubernetes/reference-content/version-support-policy.mdx [hunk 5]
```diff
@@ -42,64 +42,61 @@ Before opening a support ticket related to cluster behavior or a Kubernetes comp
...
+| 1.29               | December 2023    | February 2025        | February 7, 2024             | February 2025    | April 7, 2025*    |
+| 1.28               | August 2023      | October 2024         | August 2023                  | February 2025*   | April 7, 2025*    |
+| 1.27               | April 2023       | June 2024            | April 2023                   | February 2025*   | April 7, 2025*    |
+| 1.26               | December 2022    | February 2024        | February 6, 2023             | February 2025    | March 6, 2025     |
+| 1.25               | August 2022      | October 2023         | February 6, 2023             | February 2025    | March 6, 2025     |
 | 1.24               | April 2022       | July 2023            | April 2022                   | April 2024       | May 15, 2024      |
 | 1.23               | December 8, 2021 | February 2023        | December 9, 2021             | December 2023    | February 15, 2024 |
 
+<Message type="important">
+* New support policy: from version 1.29 release, Scaleway's support window has shifted to 14 months.
```
Comment (nerda-codes) [inline containers/kubernetes/reference-content/version-support-policy.mdx:67]:
```suggestion
* New support policy: from the release of version 1.29, Scaleway's support window has shifted to 14 months.
```

**Review by ldecarvalho-doc — APPROVED** [review]

File: containers/kubernetes/reference-content/version-support-policy.mdx (hunk 3 above)
Comment (bene2k1) [reply containers/kubernetes/reference-content/version-support-policy.mdx:91]: Graduated is the Kubernetes term for a feature that moves from alpha/beta to prod https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-graduated-or-deprecated-features

Comment (tgenaitay) [reply containers/kubernetes/reference-content/version-support-policy.mdx:91]: alpha => beta => GA, each step in between is a graduation!
I've simply switched from a numeral form, the meaning stays the same

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**